playwright-e2e/authentication/auth-dsm.ts

```diff
@@ -1,30 +1,13 @@
 import { Page } from '@playwright/test';
+import { fillEmailPassword } from 'authentication/auth-singular';
 
-// import path from 'path';
-// import * as dotenv from 'dotenv';
-// dotenv.config({ path: path.resolve(__dirname, '../dsm/.env.dsm') });
-
-export async function fillEmailPassword(
```

Comment:
Remove `fillEmailPassword()` to avoid code duplication. It is in `auth-singular.ts`.